mailet/standard/src/main/java/org/apache/james/transport/mailets/ContactExtractor.java

```diff
@@ -105,6 +109,20 @@ public void service(Mail mail) throws MessagingException {
             .map(Throwing.function(extractedContacts -> objectMapper.writeValueAsString(extractedContacts)));
     }
 
+    private MimeMessage relaxedParsableMessage(Mail mail) throws IOException, MessagingException {
+        Properties props = System.getProperties();
+        props.put("mail.mime.address.strict", false);
+        Session session = Session.getInstance(props);
```

Comment:
Can't we intruduce a SessionProvider object, that unions system properties with **mail.mime.address.strict**, that we could use everywhere, and that would avoid us doing this expensive copy?